api/vnet_error.go

```diff
@@ -0,0 +1,376 @@
+		return "System call error #1"
+	case SYSCALL_ERROR_2:
+		return "System call error #2"
+	case SYSCALL_ERROR_3:
+		return "System call error #3"
+	case SYSCALL_ERROR_4:
+		return "System call error #4"
+	case SYSCALL_ERROR_5:
+		return "System call error #5"
+	case SYSCALL_ERROR_6:
+		return "System call error #6"
+	case SYSCALL_ERROR_7:
+		return "System call error #7"
+	case SYSCALL_ERROR_8:
+		return "System call error #8"
+	case SYSCALL_ERROR_9:
+		return "System call error #9"
+	case SYSCALL_ERROR_10:
+		return "System call error #10"
+	case FEATURE_DISABLED:
+		return "Feature disabled by configuration"
+	case INVALID_REGISTRATION:
+		return "Invalid registration"
+	case NEXT_HOP_NOT_IN_FIB:
+		return "Next hop not in FIB"
+	case UNKNOWN_DESTINATION:
+		return "Unknown destination"
+	case PREFIX_MATCHES_NEXT_HOP:
+		return "Prefix matches next hop"
+	case NEXT_HOP_NOT_FOUND_MP:
+		return "Next hop not found (multipath)"
+	case NO_MATCHING_INTERFACE:
+		return "No matching interface for probe"
+	case INVALID_VLAN:
+		return "Invalid VLAN"
+	case VLAN_ALREADY_EXISTS:
+		return "VLAN subif already exists"
+	case INVALID_SRC_ADDRESS:
+		return "Invalid src address"
+	case INVALID_DST_ADDRESS:
+		return "Invalid dst address"
+	case ADDRESS_LENGTH_MISMATCH:
+		return "Address length mismatch"
+	case ADDRESS_NOT_FOUND_FOR_INTERFACE:
+		return "Address not found for interface"
+	case ADDRESS_NOT_LINK_LOCAL:
+		return "Address not link-local"
+	case IP6_NOT_ENABLED:
+		return "ip6 not enabled"
+	case IN_PROGRESS:
+		return "Operation in progress"
+	case NO_SUCH_NODE:
+		return "No such graph node"
+	case NO_SUCH_NODE2:
+		return "No such graph node #2"
+	case NO_SUCH_TABLE:
+		return "No such table"
+	case NO_SUCH_TABLE2:
+		return "No such table #2"
+	case NO_SUCH_TABLE3:
+		return "No such table #3"
+	case SUBIF_ALREADY_EXISTS:
+		return "Subinterface already exists"
+	case SUBIF_CREATE_FAILED:
+		return "Subinterface creation failed"
+	case INVALID_MEMORY_SIZE:
+		return "Invalid memory size requested"
+	case INVALID_INTERFACE:
+		return "Invalid interface"
+	case INVALID_VLAN_TAG_COUNT:
+		return "Invalid number of tags for requested operation"
+	case INVALID_ARGUMENT:
+		return "Invalid argument"
+	case UNEXPECTED_INTF_STATE:
+		return "Unexpected interface state"
+	case TUNNEL_EXIST:
+		return "Tunnel already exists"
+	case INVALID_DECAP_NEXT:
+		return "Invalid decap-next"
+	case RESPONSE_NOT_READY:
+		return "Response not ready"
+	case NOT_CONNECTED:
+		return "Not connected to the data plane"
+	case IF_ALREADY_EXISTS:
+		return "Interface already exists"
+	case BOND_SLAVE_NOT_ALLOWED:
+		return "Operation not allowed on slave of BondEthernet"
+	case VALUE_EXIST:
+		return "Value already exists"
+	case SAME_SRC_DST:
+		return "Source and destination are the same"
+	case IP6_MULTICAST_ADDRESS_NOT_PRESENT:
+		return "IP6 multicast address required"
+	case SR_POLICY_NAME_NOT_PRESENT:
+		return "Segement routing policy name required"
```

Comment:
Segment, I see they got it wrong in the VPP

Reply:
*Sigh*.. well, I'm afraid of some case when someone will try to look it up when this error occurs and won't find it because of the fixed typo.